src/parser/jobs/drg/modules/BloodOfTheDragon.js

```diff
@@ -224,10 +236,69 @@ export default class BloodOfTheDragon extends Module {
 		this._lastEvent = event.timestamp
 	}
 
+	_intersectsDowntime(start) {
+		const windows = this.downtime.getDowntimeWindows(start)
+		const end = start + DRAGON_DEFAULT_DURATION_MILLIS
+
+		for (const window of windows) {
+			if (window.start < end) {
+				return window.start
+			}
+		}
+
+		return null
+	}
+
+	_analyzeLifeWindows() {
+		for (const window of this._lifeWindows.history) {
```

Comment:
nit; aite this one scared me a tad for a second there, 'till i read the loop arg

it's _fine_ per se, but it's shadowing the global `window` object, so it'll fail pretty catastrophically if this line gets misspelled.